_data/sidebar_doc.yml

```diff
@@ -495,8 +495,11 @@ entries:
     - title: Releases
       items:
 
-        - title: v1.0-rc.2
-          url: /v1.0-rc.2.html
+        - title: 1.0
+          url: /v1.0.html
+
+        - title: Known Limitations
```

Comment:
We should put it somewhere in the sidebar, but I'd put it under FAQ instead of Releases. And maybe it should be named `known-limitations.md` instead of `v1.0-known-limitations.md`. I think we want to be able to update this page in the future by by removing things as they are fixed (and documenting the known limitations of older versions should be handled by whatever generic doc-versioning scheme we adopt).